ci/Jenkinsfile

```
@@ -0,0 +1,102 @@
+@Library('jenkins_shared_lib') _
+
+pipeline {
+  agent any
+  environment {
+    BUILD_TYPE = sh(returnStdout: true, script: 'rapids-build-type')
+  }
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '10'))
+    ansiColor('xterm')
+    disableConcurrentBuilds(abortPrevious: true)
+  }
+  stages {
+    stage("Style & Size Checks") {
+      environment {
+        GH_TOKEN = credentials('gputester-github-token')
+      }
+      when { environment name: 'BUILD_TYPE', value: 'pull-request' }
+      agent {
+        docker {
+          image "gpuci/rapidsai-driver:22.06-cuda11.5-devel-centos7-py3.8" 
```

```suggestion
          image "gpuci/rapidsai-driver:22.08-cuda11.5-devel-centos7-py3.8" 
```

Now that we've switched target branches, let's switch all the `22.06` references to `22.08` in here and in the shared library. Once we redo our CI images, they won't be tied to any particular RAPIDS version so we won't have to maintain this mapping.